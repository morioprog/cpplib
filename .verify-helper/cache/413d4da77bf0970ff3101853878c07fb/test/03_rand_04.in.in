100 313
23 8
23 52
23 73
23 7
4 97
4 11
1 38
1 39
1 75
1 48
1 86
2 29
2 50
2 78
55 29
55 66
51 66
51 28
51 30
51 6
8 18
8 44
8 40
8 69
8 60
88 25
88 28
88 12
24 18
24 22
24 61
3 46
18 15
18 78
18 37
18 58
18 82
18 60
13 80
13 10
13 11
13 99
84 9
84 46
84 10
84 41
84 57
84 81
84 89
29 79
29 73
29 47
29 91
29 54
98 9
98 10
98 30
98 77
98 57
98 49
98 16
34 46
34 93
34 22
34 58
34 5
34 11
9 25
9 96
92 97
92 36
92 7
92 58
20 25
20 32
20 63
20 75
20 96
20 70
20 33
21 43
21 27
21 90
21 19
21 42
26 56
26 77
26 48
26 87
26 82
83 75
83 48
83 42
56 52
56 38
56 73
56 37
56 65
56 64
25 74
25 30
25 77
25 91
25 53
25 95
50 15
50 17
50 81
97 32
97 63
97 37
97 48
97 64
97 49
80 40
80 37
80 58
32 78
32 94
32 90
32 71
79 40
52 75
52 49
27 85
27 62
15 22
15 60
38 39
38 66
38 6
38 37
38 19
38 87
39 60
66 87
63 7
63 14
63 19
63 87
63 95
63 86
10 6
44 78
44 30
44 14
44 58
44 17
44 64
67 19
73 28
73 40
73 91
73 19
73 42
73 89
85 76
85 47
85 96
85 5
85 17
85 49
28 16
28 11
76 96
76 0
78 90
78 62
78 33
78 45
7 12
7 48
7 91
7 65
40 77
40 71
30 75
30 5
30 72
30 86
94 77
94 14
94 91
94 61
94 64
94 53
94 99
75 62
75 72
75 49
75 95
47 81
12 77
12 48
12 11
12 71
12 86
96 62
96 33
96 19
96 64
96 81
96 99
90 77
90 58
90 69
90 5
90 65
90 59
90 16
93 5
62 70
62 45
22 48
22 16
77 19
77 61
37 91
37 65
37 49
37 11
48 69
48 33
48 17
48 87
48 81
48 54
14 41
14 91
14 19
14 17
14 42
58 41
58 5
58 49
58 45
58 60
69 81
69 95
70 59
70 99
41 59
5 82
91 57
91 59
91 42
91 64
91 11
91 54
33 17
33 61
33 16
33 95
33 54
33 99
19 86
65 59
65 49
65 89
65 99
57 17
57 81
57 95
59 11
0 11
17 42
17 64
17 53
17 71
61 87
61 53
61 16
61 89
42 87
42 53
42 60
42 54
42 99
64 82
45 87
45 82
45 53
45 16
45 89
45 86
87 82
87 16
82 53
82 60
82 99
16 60
16 71
81 11
81 60
81 89
81 95
81 54
81 99
11 89
11 71
60 89
60 95
60 86
60 54
60 99
89 99
95 71
95 86
95 54
71 54
71 99
86 54
54 99
