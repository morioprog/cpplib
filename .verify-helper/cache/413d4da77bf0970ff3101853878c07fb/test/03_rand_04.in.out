98
92
88
84
83
68
67
55
51
36
35
34
93
31
26
56
24
23
52
21
27
85
76
43
20
13
80
10
9
96
25
74
8
44
30
18
4
97
63
7
12
32
94
14
3
46
2
78
90
58
5
41
50
15
22
29
47
73
28
79
40
77
1
48
33
61
69
75
72
62
45
70
38
19
37
65
49
91
59
57
81
17
64
42
6
66
87
16
82
53
39
60
95
86
0
11
71
54
89
99
